recipes/cusparselt/meta.yaml

```
@@ -0,0 +1,93 @@
+{% set version = "0.0.1" %}
+{% set patch_version = "73" %}
+
+package:
+  name: cusparselt
+  version: {{ version }}.{{ patch_version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/libcusparse_lt/{{ version }}/local_installers/libcusparse_lt-linux-x86_64-{{ version }}.{{ patch_version }}.tar.gz   # [linux64 and cuda_compiler_version in ("10.1", "10.2", "11.0")]
+  # url: https://developer.download.nvidia.com/compute/libcusparse_lt/{{ version }}/local_installers/libcusparse_lt-linux-ppc64le-{{ version }}.{{ patch_version }}.tar.gz  # [ppc64le and cuda_compiler_version in ("10.1", "10.2", "11.0")]
+  # url: https://developer.download.nvidia.com/compute/libcusparse_lt/{{ version }}/local_installers/libcusparse_lt-linux-sbsa-{{ version }}.{{ patch_version }}.tar.gz  # [aarch64 and cuda_compiler_version in ("10.1", "10.2", "11.0")]
+  # url: https://developer.download.nvidia.com/compute/libcusparse_lt/{{ version }}/local_installers/libcusparse_lt_{{ version }}.exe  # [win64 and cuda_compiler_version in ("10.1", "10.2", "11.0")]
+
+
+  # TBD
+  sha256: 63a5a854b2df4477ae48284a23c1b7c7632c23fe418434fa70a2ff8ac238b27e  # [linux64]
+  # sha256: d914a721b8a6bbfbf4f2bdea3bb51775e5df39abc383d415b3b06bbde2a47e6e  # [ppc64le]
+  # sha256: 031b75946d2ce22f8c2736f7b4216b9377406afcb35012093a66e2801bfe2990  # [aarch64]
+  # sha256: 88d3e2c662e601214ef5caa26ca2db0492eab6ec3bf9dc9a746bb1a7b8c3aab2  # [win64]
+
+build:
+  number: 0
+  skip: True  # [win32 or cuda_compiler_version in ("None", "9.2", "10.0")]
```

Sorry, I didn't know this is 11.0+ only. In this case, we need to expand the skip condition. But, since we are using 11.0 to support all 11.x, it's actually simpler:
```suggestion
  skip: True  # [not linux or cuda_compiler_version != "11.0")]
```